README.md

```diff
@@ -776,11 +775,12 @@ Currently, DocArray supports the following vector databases:
 - [Qdrant](https://qdrant.tech/)
 - [Elasticsearch](https://www.elastic.co/elasticsearch/) v8 and v7
 - [Redis](https://redis.io/)
-- [HNSWlib](https://github.com/nmslib/hnswlib) as a local-first alternative
+- ExactNNMemorySearch as a local alternative with exact kNN search.
+- [HNSWlib](https://github.com/nmslib/hnswlib) as a local-first ANN alternative
 
 An integration of [OpenSearch](https://opensearch.org/) is currently in progress.
 
-DocArray <=0.21 also support [Redis](https://redis.io/) and [Milvus](https://milvus.io/), but these are not yet supported in the current version.
+DocArray <=0.21 also support [Milvus](https://milvus.io/), but this is not yet supported in the current version.
```

Comment:
we should remove this line completely now

Reply:
wait until it is released the Milvus integration